load-tests/user-files/simulations/NtbsLoadTest.scala

```diff
@@ -0,0 +1,108 @@
+import scala.concurrent.duration._
+import scala.util.Random
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.jdbc.Predef._
+
+class NtbsLoadTest extends Simulation {
+
+    val httpProtocol = http
+        .baseUrl("https://ntbs-load-test.e32846b1ddf0432eb63f.northeurope.aksapp.io")
+        .inferHtmlResources(WhiteList("""https://ntbs-load-test.e32846b1ddf0432eb63f.northeurope.aksapp.io/.*"""), BlackList())
+
+    val notificationFeeder = Iterator.continually(Map("notificationId" -> (300001 + Random.nextInt(10000))))
+
+    val dashboard = DashboardScenarioBuilder.build()
+    val searchByFamilyName = SearchScenarioBuilder.build(familyName = "Test")
+    val searchById = SearchScenarioBuilder.build(id = "${notificationId}")
```

Comment:
I've just noticed that we're not doing any searches by partial date, which is something mentioned in the ACs so I'll have to add that